cats/__init__.py

```diff
@@ -219,7 +218,7 @@ def main(arguments=None):
             "      specify the scheduler with the -s or --scheduler option"
         )
         sys.exit(1)
-    config, CI_API_interface, location, duration = get_runtime_config(args)
+    CI_API_interface, location, duration, jobinfo, PUE = get_runtime_config(args)
```

Comment:
The `config` dict representing the yaml config file can now be confined to the `configure` module and not appear in the `__init__` module. The `jobinfo` list of `(ndevice, power)` is all that is require for carbon footprint estimation down the line.